From ce9c6c50fe9d10155842fdee4f911656992bf76b Mon Sep 17 00:00:00 2001
From: Bart Massey <bart@cs.pdx.edu>
Date: Mon, 21 Sep 2009 11:36:15 -0700
Subject: Staging: comedi: ni_labpc_cs: added mandatory whitespace to comparison


should probably replace this code, no need to duplicate what is already
in the main kernel tree.



Signed-off-by: Bart Massey <bart@cs.pdx.edu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/comedi/drivers/ni_labpc_cs.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/comedi/drivers/ni_labpc_cs.c
+++ b/drivers/staging/comedi/drivers/ni_labpc_cs.c
@@ -162,7 +162,7 @@ static int labpc_attach(struct comedi_de
 #ifdef PCMCIA_DEBUG
 static int pc_debug = PCMCIA_DEBUG;
 module_param(pc_debug, int, 0644);
-#define DEBUG(n, args...) if (pc_debug>(n)) printk(KERN_DEBUG args)
+#define DEBUG(n, args...) if (pc_debug > (n)) printk(KERN_DEBUG args)
 static const char *version =
     "ni_labpc.c, based on dummy_cs.c 1.31 2001/08/24 12:13:13";
 #else
